### ISSUE TITLE
Overfitting in Pocket Fine-Tuning

### ISSUE THREAD

**Issue by kkkayle** [jkwang93/Token-Mol#3]
Hello,

Thank you very much for sharing such innovative work. While following the steps in the README to reproduce your results, I encountered an issue where the fine-tuning task on the pocket dataset began to overfit by the second epoch. Additionally, it seems that early stopping should not be implemented within the evaluation function. Could you please let me know if this behavior is expected?

Furthermore, when I used the fine-tuned results from the first epoch to test with test_protein_represent.pkl, I found that the results significantly differed from those reported in the paper. Could this discrepancy be due to differences between the test set used in the paper and the one provided in the repository?

I look forward to your response!

**Comment by jkwang93** [comment]
Thank you very much for your attention to our work. Responses on GitHub might not be very prompt, so if you have any questions, feel free to contact us via email for a faster reply: jikewang@zju.edu.cn. We did not encounter the overfitting issue you mentioned. In the updated version of the code, we have provided fine-tuned model, which you can directly use. Additionally, if you need the generated molecules from the evaluations in the paper, we can provide them as well.
